composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "naugrim/bmecat",
     "type": "library",
+    "varsion": "1.0.0.1",
```

Comment:
typo here
but actually the version should come from the git tag anyway, so i think this can be removed